### ISSUE TITLE
NVDA screen reader slowdown (Windows)

### ISSUE THREAD

**Issue by mkrautz** [mumble-voip/mumble#976]
DomasoFan reports on IRC that the NVDA screen reader is slow at reading Mumble's UI using the default settings on Windows.

When disabling winhooks, or launching NVDA after Mumble, the slowness is cured.

_This ticket has been migrated from sourceforge. It is thus missing some details like original creator etc.
The original is at https://sourceforge.net/p/mumble/bugs/976/ ._

**Comment by tapper82** [comment]
Could some one pleas fix this pleas?

**Comment by dingpengyu** [comment]
The problem still exists in 1.3.0

**Comment by dingpengyu** [comment]
cc @davidebeatrici 
thank

**Comment by davidebeatrici** [comment]
Fixed in #3896.
